src/csm_testing/tests/iuf_stages_cleanup/__main__.py

```diff
@@ -0,0 +1,187 @@
+#
+# MIT License
+#
+# (C) Copyright 2024 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+"""
+This script will cleanup the all the data uploaded by IUF run of dummy-product.
+"""
+import base64
+import os
+import subprocess
+from requests import Session
+import requests
+from requests.auth import HTTPBasicAuth
+from kubernetes import client, config
+import sys
+
+IMAGE = "registry.local/artifactory.algol60.net/csm-docker/stable/product-deletion-utility:1.0.1"
+NAMESPACE = "services"
+CONFIGMAP_NAME = "cray-product-catalog"
+PRODUCT_NAME = "dummy"
+PRODUCT_VERSION = "1.0.0"
+DELETION_FILE_PATH="/etc/cray/upgrade/csm/iuf/deletion"
+NEXUS_URL = "https://packages.local"
+REPO_URL = f"{NEXUS_URL}/service/rest/v1/repositories/yum/hosted"
```

Comment:
Most of these fields are also defined in the `iuf_stages_cleanup` test. I am guessing that if you change them in one test, you'd also need to change them in the other test. That means they should probably be moved into a separate module that is shared and imported by both tests.